tests/integration_tests/test_lightgbm.py

```diff
@@ -31,51 +41,58 @@ def test_lightgbm_pruning_callback_call():
         pruning_callback(env)
 
 
-def test_lightgbm_pruning_callback():
-    # type: () -> None
+@pytest.mark.parametrize('cv', CV_FLAGS)
+def test_lightgbm_pruning_callback(cv):
+    # type: (bool) -> None
 
     study = optuna.create_study(pruner=DeterministicPruner(True))
-    study.optimize(objective, n_trials=1)
+    study.optimize(partial(objective, cv=cv), n_trials=1)
     assert study.trials[0].state == optuna.structs.TrialState.PRUNED
 
     study = optuna.create_study(pruner=DeterministicPruner(False))
-    study.optimize(objective, n_trials=1)
+    study.optimize(partial(objective, cv=cv), n_trials=1)
     assert study.trials[0].state == optuna.structs.TrialState.COMPLETE
     assert study.trials[0].value == 1.
 
     # Use non default validation name.
     custom_valid_name = 'my_validation'
     study = optuna.create_study(pruner=DeterministicPruner(False))
-    study.optimize(lambda trial: objective(trial, valid_name=custom_valid_name), n_trials=1)
+    study.optimize(lambda trial: objective(trial, valid_name=custom_valid_name, cv=cv),
+                   n_trials=1)
     assert study.trials[0].state == optuna.structs.TrialState.COMPLETE
     assert study.trials[0].value == 1.
 
 
-def test_lightgbm_pruning_callback_errors():
-    # type: () -> None
+@pytest.mark.parametrize('cv', CV_FLAGS)
+def test_lightgbm_pruning_callback_errors(cv):
+    # type: (bool) -> None
 
     # "maximize" direction isn't supported yet.
     study = optuna.create_study(pruner=DeterministicPruner(False))
     with pytest.raises(ValueError):
-        study.optimize(lambda trial: objective(trial, metric='auc'), n_trials=1, catch=())
+        study.optimize(lambda trial: objective(trial, metric='auc', cv=cv), n_trials=1, catch=())
 
     # Unknown metric
     study = optuna.create_study(pruner=DeterministicPruner(False))
     with pytest.raises(ValueError):
-        study.optimize(lambda trial: objective(trial, metric='foo_metric'), n_trials=1, catch=())
-
-    # Unknown validation name
-    study = optuna.create_study(pruner=DeterministicPruner(False))
-    with pytest.raises(ValueError):
-        study.optimize(lambda trial: objective(trial, valid_name='valid_1',
-                                               force_default_valid_names=True),
+        study.optimize(lambda trial: objective(trial, metric='foo_metric', cv=cv),
                        n_trials=1, catch=())
 
+    if not cv:
+        # Unknown validation name
+        study = optuna.create_study(pruner=DeterministicPruner(False))
+        with pytest.raises(ValueError):
+            # `'valid_name'` will be ignored if `cv == True`, since it will be fixed to `'agg_cv'``
```

Comment:
[nits] I think this line (comment) can be removed because the current code doesn't test the case where `cv == True`.

Reply:
Thank you for your notice.  It seems that I forgot to remove the comment.
Now I removed the comment.